sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -156,3 +156,10 @@ export async function createCustomTestProject(
     .put(deployParam);
   await polling(client, deployTask);
 }
+
+export function isSovereignClouds(endpoint: string): boolean {
+  if (endpoint.includes(".us") && endpoint.includes(".cn")) {
```

Comment:
```suggestion
  if (endpoint.includes(".us") || endpoint.includes(".cn")) {
```

Reply:
We have updated the file and this has been fixed. Thank you!